### PR TITLE
impl builtin:func to_base64()

### PR DESCRIPTION
**Pull request by woodpenker**
Sorry I delete the fork.So I have to create a new PR.. :sob: 
This PR refer to [#3055](https://github.com/pingcap/tidb/pull/3055)

### CODE REVIEW

**Comment by tiancaiamao** [comment]
LGTM

File: expression/builtin_string.go [hunk 1]
```diff
@@ -1771,6 +1771,8 @@ func (b *builtinFromBase64Sig) eval(row []types.Datum) (d types.Datum, err error
 	if err != nil {
 		return d, errors.Trace(err)
 	}
+	str = strings.Replace(str, "\t", "", -1)
```
Comment (hanfei1991) [inline expression/builtin_string.go]: We need to add test for it ?

Comment (woodpenker) [inline expression/builtin_string.go]: I already add it in this:
``` go
{
 +			string("QUJDREVGR0hJSkt\tMTU5PUFFSU1RVVld\nYWVphYmNkZ\rWZnaGlqa2xt   bm9wcXJzdHV2d3h5ejAxMjM0NTY3ODkrLw=="),
 +			string("ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/"),
 +		},
```
you can see there are \r \t in it

Comment (hanfei1991) [inline expression/builtin_string.go]: ok

**Comment by hanfei1991** [comment]
LGTM

**Pull request merged**